app/javascript/components/users/show/_InfoHelper.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const InfoHelper = ({ name, value, link}) => {
+  return (
+    <>
+    <div className='user-item'>
+      <span className='naming-user'>{ name }</span>
+      <span className='insert-user'>{ value? value : 'Unknown'  } { link }</span>
```

Comment:
`value? value : 'Unknown'` is it ternary operator?, if yes needs space before question mark